src/lando/utils/management/commands/create_environment_repos.py

```diff
@@ -168,6 +168,22 @@
         }
     )
 
+# Set up `infra-testing-*` repos in prod.
+# See bug 1966728.
+for branch in ["release", "beta", "esr128", "esr115"]:
```

Comment:
Nit: Can we add `autoland` as well? It's been manually created, but we might as well be exhaustive.

```suggestion
for branch in ["autoland", "release", "beta", "esr128", "esr115"]:
```